### ISSUE TITLE
[Bug][Installation] lack of `res.upload.startup.type` in `install.sh`

### ISSUE THREAD

**Issue by erienniu** [apache/incubator-dolphinscheduler#4073]
**Which version of Dolphin Scheduler:**
 -[1.3.3]

there's no property `res.upload.startup.type` but `resource.storage.type` in `install.sh` and `install_config.conf`, so upoad is invalidable after deploy and cause error belows

```
[ERROR] 2020-11-17 14:10:13.520 org.apache.dolphinscheduler.api.exceptions.ApiExceptionHandler:[45] - 创建资源错误
java.lang.IllegalArgumentException: No enum constant org.apache.dolphinscheduler.common.enums.ResUploadType.
	at java.lang.Enum.valueOf(Enum.java:238)
	at org.apache.dolphinscheduler.common.enums.ResUploadType.valueOf(ResUploadType.java:22)
	at org.apache.dolphinscheduler.common.utils.PropertyUtils.getResUploadStartupState(PropertyUtils.java:76)
	at org.apache.dolphinscheduler.api.service.ResourcesService.createDirectory(ResourcesService.java:101)
	at org.apache.dolphinscheduler.api.service.ResourcesService$$FastClassBySpringCGLIB$$cfb54838.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:218)
        ……
```

**Comment by zhuangchong** [comment]
'res.upload.startup.type' has not been found in the 1.3.3 branch code. Please check the 'resource.storage.type' attribute in the common.properties file under the dolphin installation path.

---
'res.upload.startup.type' 在1.3.3分支代码里面没有查到系统引用的地方,请查看dolphin安装路径下common.properties 文件内'resource.storage.type'属性写的是什么？

![image](https://user-images.githubusercontent.com/37063904/99475139-e0e00180-2988-11eb-92a3-8436523a1ec8.png)

**Comment by nzy0x00** [comment]
1.3.3-release，我用dockerfile构建的镜像，发现common.properties 文件里的resource.storage.type的属性为空（不是NONE，就是空白），创建租户时报错No enum constant org.apache.dolphinscheduler.common.enums.ResUploadType。
把resource.storage.type配置为NONE之后就可以成功运行。

**Comment by zhuangchong** [comment]
ok, the 1.3.3-release branch did not perform default value processing on the configuration file built by docker, and the dev branch has done related processing
@dailidong @CalvinKirs See if it needs to be added to docker-release Milestones

---
好的，1.3.3-release分支没有对docker 构建的配置文件做默认值处理，dev分支有做过相关处理
@dailidong @CalvinKirs 看一下是否需要加到docker-release  Milestones

**Comment by dailidong** [comment]
> 分支有做过相关处理

please add it to 1.3.4 milestone

**Comment by lgcareer** [comment]
@nzy0x00 Hi,Which command you use to run the docker?docker or docker-compose?I use docker-compose to start and the default value of resource.storage.type is HDFS.

**Comment by nzy0x00** [comment]
@lgcareer  Hi, I use docker.

**Comment by lgcareer** [comment]
@nzy0x00 I will fix it in the 1.3.4-release,In the 1.3.3 branch you need set `-e DOLPHINSCHEDULER_RESOURCE_STORAGE_TYPE="HDFS"` when you exectute `docker run`

**Comment by nzy0x00** [comment]
@lgcareer  OK，thank u.
